aodntools/timeseries_products/velocity_aggregated_timeseries.py

```diff
@@ -137,7 +75,7 @@ def velocity_aggregated(files_to_agg, site_code, input_dir='', output_dir='./',
     n_obs_total = 0
     for file in files_to_agg:
         with xr.open_dataset(os.path.join(input_dir, file)) as nc:
-            error_list = check_file(nc, site_code)
+            error_list = check_file(nc, site_code, 'velocity')
```

Comment:
See previous comment about the velocity argument. Note the varlist is defined in this function but not used anymore.